packages/react-select/src/Select.tsx

```diff
@@ -441,6 +449,31 @@ function buildFocusableOptionsFromCategorizedOptions<
   );
 }
 
+function buildFocusableOptionsWithIds<Option, Group extends GroupBase<Option>>(
```

Comment:
This function calculates ids for focusable options as id is needed in 'aria-activedescendant'.